model/epoch.ts

```diff
@@ -1,5 +1,5 @@
 export type Epoch = {
-  id: bigint
-  startTimestamp: bigint
-  endTimestamp: bigint
+  id: number
+  startTimestamp: number
+  endTimestamp: number
```

Comment:
Why change this to a number type?

Reply:
`number` is more natural as it isn't used to hold a `uint256` size value. Also next.js doesn't allow bigint since it can't be serialized into a json.